src/models/server.ts

```diff
@@ -18,4 +19,9 @@ const ServerSchema = new Schema<IServer, ServerModel, IServerMethods>({
 
 ServerSchema.plugin(uniqueValidator, { message: '{PATH} already in use.' });
 
-export const Server = model<IServer, ServerModel>('Server', ServerSchema);
\ No newline at end of file
+export const Server = model<IServer, ServerModel>('Server', ServerSchema);
+
+export const serverDeviceToSystemType: Record<number, SystemType> = {
+	1: 'WIIU',
+	2: '3DS'
+};
```

Comment:
The `models` folder is really only supposed to be used for Mongoose-specific code. I'm not sure this is the best place for this (for example if the use of strings was dropped in favor of number enums like it was before, then that would more nicely fit in the `types` folder)

Reply:
I'll move it to types 👍